samples/dashboards/data/mqtt-connector/demo.css

```diff
@@ -0,0 +1,138 @@
+@import url("https://code.highcharts.com/css/highcharts.css");
+@import url("https://code.highcharts.com/dashboards/css/datagrid.css");
+@import url("https://code.highcharts.com/dashboards/css/dashboards.css");
+
+:root {
+    /* Dimensions */
+    --chart-height: 200px;
+    --log-height: 250px;
+
+    /* Colors */
+    --custom-background: var(--highcharts-neutral-color-3);
+}
+
+/* *
+ *
+ *  Layout
+ *
+ * */
+
+.row {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+}
+
+.cell {
+    flex: 1;
+    min-width: 20px;
+}
+
+.cell > .highcharts-dashboards-component {
+    position: relative;
+    margin: 10px;
+    padding: 10px;
+}
+
+/* *
+ *
+ *  Dashboards/Datagrid specific
+ *
+ * */
+.highcharts-dashboards-wrapper {
+    background-color: var(--highcharts-neutral-color-10);
+    padding: 10px;
+    overflow: hidden;
+}
+
+.highcharts-dashboards-component {
+    background-color: var(--custom-background);
+    max-height: var(--chart-height);
+}
+
+.highcharts-datagrid-container {
+    font-size: 12px;
```

Comment:
Can we use `1em` instead? We will not limit zooming in, so the accessibility will be on a higher level.

Reply:
Please replace px with em everywhere.